parlai/agents/language_model/language_model.py

```diff
@@ -574,7 +578,18 @@ def receive_metrics(self, metrics_dict):
         if 'loss' in metrics_dict and self.scheduler is not None:
             self.scheduler.step(metrics_dict['loss'])
 
-    def load(self, path):
-        """Return opt and model states."""
+    def load_opt(self, path):
+        """Return opt, states."""
         states = torch.load(path, map_location=lambda cpu, _: cpu)
-        return states['opt'], states
+        return states['opt']
+
+    def load(self, path):
+        """Load model states."""
+        if os.path.isfile(path):
+            # load model parameters if available
+            print('[ Loading existing model params from {} ]'.format(path))
+            self.states = torch.load(path, map_location=lambda cpu, _: cpu)
+
+        if self.states:
```

Comment:
can be in same if above?

Reply:
not sure what you mean by this

Reply:
oh ok i see the load might fail so states wouldnt be set.. should that error or ?

Reply:
oh i see what you're saying. i don't think that should ever happen. will get rid of it.